### DISCUSSION TITLE
Themed Events

### DISCUSSION THREAD

**Discussion by msmichellegar** [lnug/feedback#111] [Ideas]
I'd like to propose a new way of organising talks at LNUG, which I think could help focus people's creativity as they think about submitting talks, and would also allow for some more specific outreach to potential speakers.

In the past, LNUG has done "themed" events -- I'm thinking of the functional programming edition in 2016/2017. Quite a few other meetups do this successfully. Last night I went to a really great [We Love Tech event](https://ti.to/we-love-tech/we-love-accessibility/with/uoakop4kc1o) which had a narrow focus on accessibility for the evening that allowed for some really niche, interesting talks.

What do you think about experimenting with this for LNUG? We could try it for three months and see how it goes. We could announce all three themes a couple of months in advance, and scout related talks.

Here are some themes I thought of. Would love to hear more creative themes, or themes based around technologies that people are particularly hyped about right now.
* GraphQL
* Breaking Production: stories of things going wrong and lessons learned
* Machine learning with JS
* Useless things showcase: people share useless JavaScript side projects they've built (inspired by this: http://www.stupidhackathon.com/)

**Comment by claireinez** [comment]
Love this idea, Michelle! I wasn't able to make it to the FP one but heard it was one of the most popular LNUG events yet. We should definitely be scouting talks; I've done it in bursts in the past but found it quite difficult without a focus but a theme makes it slightly easier to do that.

Another theme could be "Using universal JS in your apps".

**Comment by admataz** [comment]
Hey @msmichellegar  - this is a great idea, thank you for raising this. I'd love to do a themed night or more for LNUG. Yes - it could be a great way to give the meetups focus and find relevant speakers. A similar approach has been discussed at various points among organisers - I think you'll find it well supported! 

And I agree, currently we rely too heavily on speakers volunteering talks - it's great to hear some ideas coming forward for fresh approaches. 

The FP evening did go very well, although that was more on part of the speakers who approached us with the idea of three speakers on the theme  - and they worked together to coordinate the topics and curation and programming of evening's talks. So LNUG just provided the platform - but we could take some learnings on how they approached the curation of the evening (they thought very carefully about how they introduced and progressed into more advanced and applied FP concepts through as the talks progressed)

In terms of special topics and themed events, some more ideas from me...
 - node.js in media (Condé Nast, FT, The Times... regular LNUG attendees and contributors)
 - node.js for IoT (both hobbiests/amateur roboticists, and commercial applications like next month's speaker from EVRYTHNG)
 - node.js as enabler for startups  - and beyond

**Comment by clarkie** [comment]
We've talked about this in the past but have struggled to get enough momentum. Lining up a bunch of speakers to talk on the same topic on the same night can be tricky but let's give it a go. Another one from me:

- Music & JS, the perfect composition!

What's the best approach to getting more people involved? Shall we create the themes as "labels" and start tagging talk proposals if they fit?

**Comment by msmichellegar** [comment]
Thanks, that's excellent feedback @admataz @clarkie. Really love your suggestions! Some of those I can definitely see us being able to encourage proposals for from attendees (media and music).

How about choosing 3-4 of these, and putting them on a slide for the next LNUG? Whoever is MC-ing could give some context and ask for submissions/recommendations around these topics for August/September onwards. The order in which the themes are covered can depend on submissions and talks we're able to source.

Also would be great to tag proposals like you suggest @clarkie ✅

**Comment by orliesaurus** [comment]
+1 clarkie's idea an OP's suggestion

**Comment by msmichellegar** [comment]
Another suggestion: internationalisation.

How do we feel about making this happen? @admataz @clarkie 

If so, I propose we do GraphQL for September as I will have a talk by then :) Happy to help scout speakers for that month.

**Comment by admataz** [comment]
Hi @msmichellegar  - sounds great! - let's make GraphQL our first themed event for September and tr y out this approach. Can you please put in an initial talk proposal as a placeholder that we can put into the schedule for September (no need for it to finalised at this point)
